administrator-guides/authentication/ldap/README.md

```diff
@@ -19,11 +19,12 @@ For now (until we add more input fields to LDAP) set it like this: (This is base
 - LDAP_Port = 389
 - LDAP_Bind_Search =
 
-{"filter": "(&(objectCategory=person)(objectclass=user)(memberOf=CN=ROCKET_ACCESS,CN=Users,DC=domain,DC=com)(sAMAccountName=#{username}))", "scope": "sub", "userDN": "rocket.service@domain.com", "password": "urpass"}
+`{"filter": "(&(objectCategory=person)(objectclass=user)(memberOf=CN=ROCKET_ACCESS,CN=Users,DC=domain,DC=com)(sAMAccountName=#{username}))", "scope": "sub", "userDN": "rocket.service@domain.com", "password": "urpass"}`
 
 If you need to auth users from subgroups in LDAP use this filter:
 
-- LDAP_Bind_search = {"filter": "(&(objectCategory=person)(objectclass=user)(memberOf:1.2.840.113556.1.4.1941:=CN=ROCKET_ACCESS,CN=Users,DC=domain,DC=com)(sAMAccountName=#{username}))", "scope": "sub", "userDN": "rocket.service@domain.com", "password": "urpass"}
+- LDAP_Bind_search = 
```

Comment:
@paulofariadias: Fix Travis CI errors

- Remove any trailing spaces
- Make sure there's a space between the start of the list and the previous line.